app/src/main/java/com/bignerdranch/android/playlistmaker/playlist_menu/ui/fragment/PlaylistMenuFragment.kt

```diff
@@ -205,14 +217,14 @@ class PlaylistMenuFragment : Fragment() {
         }
     }
 
+    @SuppressLint("SuspiciousIndentation")
     private fun showMessage() {
         val message = getString(R.string.empty_track_list)
-        Snackbar
-            .make(requireContext(), binding.containerLayout, message, Snackbar.LENGTH_SHORT)
-            .setBackgroundTint(ContextCompat.getColor(requireContext(), R.color.blue))
-            .setTextColor(ContextCompat.getColor(requireContext(), R.color.white))
-            .setDuration(MESSAGE_DURATION_MILLIS)
-            .show()
+        snackbar = Snackbar.make(requireContext(), binding.containerLayout, message, Snackbar.LENGTH_SHORT)
+            snackbar!!.setBackgroundTint(ContextCompat.getColor(requireContext(), R.color.blue))
+            snackbar!!.setTextColor(ContextCompat.getColor(requireContext(), R.color.white))
+            snackbar!!.setDuration(MESSAGE_DURATION_MILLIS)
+            snackbar!!.show()
```

Comment:
🍏 Вместо создания риска падения приложения в случае, если snackbar == null, сделай так:
```suggestion
Snackbar.make(requireContext(), binding.containerLayout, message, Snackbar.LENGTH_SHORT)
            .apply {
                setBackgroundTint(ContextCompat.getColor(requireContext(), R.color.blue))
                setTextColor(ContextCompat.getColor(requireContext(), R.color.white))
                duration = MESSAGE_DURATION_MILLIS
            }
            .show()
```